app/models/keyword.rb

```diff
@@ -2,4 +2,7 @@
 
 class Keyword < ApplicationRecord
   belongs_to :user
+
```

Comment:
we could set enum for the `status` https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html

Reply:
Fixed in ba441144c8110a345f1162715b927493464d3bad